gmso/tests/test_atom_type.py

```diff
@@ -16,6 +16,12 @@ class TestAtomType(BaseTest):
     def atomtype_metadata(self):
         return AtomType()
 
+    def test_atom_type_tag_kwarg(self):
+        at = AtomType(
+            tags={"element": "Li", "comesFrom": "ForceFieldExperiments"}
+        )
+        assert at.tag_names == ["element", "comesFrom"]
+
```

Comment:
Should we also add a test that ensures that the tags, if empty, return an empty list?